src/libcore/platforms/react-native.js

```diff
@@ -0,0 +1,211 @@
+// @flow
+
+import { NotEnoughBalance, NetworkDown } from "@ledgerhq/errors/lib";
+import type { Core, CoreStatics } from "../types";
+import { reflect } from "../types";
+import { setLoadCoreImplementation } from "../access";
+import { setRemapLibcoreErrorsImplementation } from "../errors";
+
+export default (arg: { getNativeModule: (id: string) => any }) => {
+  const { getNativeModule } = arg;
+
+  async function loadCore(): Promise<Core> {
+    // Abstract out the libcore interface into a higher level API.
+
+    const mappings: Object = {};
+
+    const wrappers = {
+      // FIXME need to fix it in RN bindings
+      hex: value => value.replace(/[< >]/g, "")
+    };
+
+    function wrap(id, ref) {
+      if (!ref) return ref;
+      if (Array.isArray(id)) {
+        const [actualId] = id;
+        return ref.map(r => wrap(actualId, r));
+      }
+      if (id in wrappers) {
+        return wrappers[id](ref);
+      }
+      const Clz = mappings[id];
+      if (!Clz) {
+        console.warn(`Unkown wrapping '${id}'`);
+        return ref;
+      }
+      return new Clz(ref);
+    }
+
+    function unwrap(id, instance, ctx) {
+      if (!instance) return instance;
```

Comment:
same here ? why do you return it if it doesn't exists/ is falsy ?

Reply:
same yes.

in that case it is to address optional parameters.